lua/wikis/marvelrivals/Infobox/Character/Custom.lua

```diff
@@ -51,13 +93,26 @@ function CustomInjector:parse(id, widgets)
 	return widgets
 end
 
+---@return Widget[]
+function CustomHero:_getRole(args)
+    local role = (args.role or ''):lower()
+    local roleLookup = ROLE_LOOKUP[role]
+
+	if roleLookup then
+		return roleLookup
+	else
+		return { 'NPC' }
+	end
+end
+
 ---@param lpdbData table
 ---@param args table
 function CustomHero:addToLpdb(lpdbData, args)
-	lpdbData.information = args.name
-	lpdbData.image = args.image
-	lpdbData.date = args.released
+	lpdbData.information = 'Playable Character'
```

Comment:
What is this needed for?

Reply:
Currently we are only tracking the characters that are playable in game (the heroes). However there are a host of NPC characters in which we can/will add in the near future. This was added to be a sort of tracking tack that can allow us to filter the characters by whether they were playable or not. Initially i was going to put this flag in Extradata, but considering its directly relevant to the heroes information, went with the flag being in information